ui/js/pods-dfv/pods-dfv.min.js

```diff
@@ -1145,7 +1145,7 @@ var PodsDFV = (function () {
   		this.showChildView('list', list);
 
   		// Show Add New?
-  		if (this.fieldConfig.get('iframe_src') !== '') {
+  		if (this.fieldConfig.get('iframe_src') !== '' && this.fieldConfig.get('pick_allow_add_new') == 1) {
```

Comment:
We should usually strive to keep comparisons as yoda conditions. But I'm okay if this goes in because the condition before it isn't.